src/ARToolkit/ARSceneView.cs

```diff
@@ -271,7 +271,7 @@ public Mapping.Camera OriginCamera
             get => _controller?.OriginCamera ?? throw new InvalidOperationException();  // Only null in design mode
             set
             {
-                if (_controller != null && value != OriginCamera && value?.IsEqual(OriginCamera) == false)
+                if (_controller != null && value != OriginCamera)
```

Comment:
I've tested on iOS and Android and both work. This implementation matches native iOS and native Android toolkits. Using `IsEqual` isn't a great option since it was deprecated.